index.ts

```diff
@@ -42,7 +42,11 @@ export {
   LiveResponse,
   LiveValidationIssue,
   LiveValidatorOptions,
-  ApiOperationIdentifier
+  ApiOperationIdentifier,
+  RequestValidationResult,
+  ResponseValidationResult,
+  ValidateOptions,
+  formatUrlToExpectedFormat
```

Comment:
I don't think we want to export "formatUrlToExpectedFormat" thats a function internal to the module

Reply:
Removed